src/styles/_transition.scss

```
@@ -35,3 +35,78 @@ $sf: 'slide-fade';
 .#{$sf}-move {
   transition: transform 3s;
 }
+
+$sv: 'slide-vertical';
+.#{$sv}-enter-active,
+.#{$sv}-leave-active {
+  transition: all 0.1s linear;
```

ちょっと`linear`はぎこちない感じになってしまうので、座標変化には使いたくないです
最初の動きが重いのもいやなので、最後に減速する形ということでease outとかどうでしょうか
```suggestion
  transition: all 0.15s $easeOutQuad;
```

---

あ～、そのほうがよいですね 👍